### PR TITLE
Improve charity search

### PR DESCRIPTION
**Pull request by ArmintasP**
- When looking for charities, all charities are shown if no category is selected.
- After selecting a category, page loads content using ajax instead of redirecting to another view.

### CODE REVIEW

File: Karma/Views/Home/SelectCategory.cshtml [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
+<script>
+    $(document).ready(function () {
+        Get();
+    });
+
+    function Get() {
+        var e = document.getElementById("category");
+        var category = e.value;
+
+        console.log("was");
```
Comment (PauliusLu) [inline Karma/Views/Home/SelectCategory.cshtml:28]: Was it only for debug, or is this somehow necessary?

**Review by PauliusLu — APPROVED** [review]
lgtm

File: Karma/Views/Home/SelectCategory.cshtml (hunk 1 above)
Comment (ArmintasP) [reply Karma/Views/Home/SelectCategory.cshtml]: Thanks, I removed this junk from the code.

**Review by baikauskaite — APPROVED** [review]
Looks good to me

**Pull request merged**